### ISSUE TITLE
Turn lanes: Wrong arrow highlighted

### ISSUE THREAD

**Issue by scaidermern** [osmandapp/OsmAnd#11177]
In the turn lanes widget OsmAnd highlights the _turn-right_ arrow although it should be the _through_ arrow instead:

![photo_2021-03-17_10-34-08](https://user-images.githubusercontent.com/1040784/111446418-ac007300-870c-11eb-9ede-26a17d5c7577.jpg)

From: 51.0341888,13.8233328
To: 51.0334331,13.8236332

[route.gpx](https://github.com/osmandapp/OsmAnd/files/6155238/route.gpx.txt)

Tagging is correct as far as I can see.

**Comment by vshcherb** [comment]
Very tricky case actually - https://www.openstreetmap.org/way/384429680

**Comment by scaidermern** [comment]
> Very tricky case actually - https://www.openstreetmap.org/way/384429680

True. However before this there is a real right turn which is _not_ to be taken. So based on this information, shouldn't it be possible for OsmAnd to remove the right turn arrow from the list of possible choices? I have no idea how the implementation looks like.

**Comment by vshcherb** [comment]
Yes but it is so sharp right turn that looks like right u-turn and more problematically next way segment has completely different meaning for "straight;right turn" which confuses algorithm so much.

**Comment by pl71** [comment]
The subject lane indicator is for the first junction (node 3061333822).
Street is tagged with "turn:lanes:forward=left|through;right".
This is what we can see from the Bing imagery also painted on the road.
I suppose "left" and "right" are for Wehlener Straße (main road pictured) in respective directions, since "through" is for continuing on Salbachstraße (real destination in this topic). 
From aerial imagery you can see Salbachstraße has a bend, leading near the subject junction, so "through" should be assumed that way by drivers.
First problem: we have a [junction ](https://www.openstreetmap.org/node/3061333822) with 2 exits in OSM, but with 3 destinations tagged.
Second: OA probably is expecting "sharp_right" for every turn exceeding 90 deg. 
Tagging should be "turn:lanes:forward=left|through;sharp_right", though there is no proper [wiki ](https://wiki.openstreetmap.org/wiki/Key:turn) example.
